app/views/wallet/TransactionView/SendMobPanel/SendMobForm.tsx

```diff
@@ -150,6 +153,33 @@ const SendMobForm: FC = () => {
     },
   ];
 
+  const handleChecked = () => {
+    setIsChecked(!isChecked);
+  };
+
+  const saveToContacts = async (alias: string, recipientAddress: string) => {
+    const randomColor = () => {
+      const RANDOM_COLORS = ['#8B35E0', '#1F639A', '#EAA520', '#15A389', '#8D969D', '#D82E26'];
+      return RANDOM_COLORS[Math.floor(RANDOM_COLORS.length * Math.random())];
+    };
+
+    const result = await assignAddressForAccount({
+      accountId: selectedAccount.account.accountId,
+    });
+
+    contacts.push({
+      abbreviation: alias[0],
```

Comment:
This will crash if the `alias` is an empty string.

Reply:
Submit is now disabled whenever save to contacts is checked and alias is an empty string:
<img width="832" alt="Screen Shot 2021-04-14 at 6 44 01 PM" src="https://user-images.githubusercontent.com/22161142/114794849-e2a0cc00-9d52-11eb-81ed-8624eefdc81d.png">
<img width="832" alt="Screen Shot 2021-04-14 at 6 44 13 PM" src="https://user-images.githubusercontent.com/22161142/114794855-e6345300-9d52-11eb-9503-6e320cb923a9.png">